prettier.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: [
+    './index',
+    'prettier-stylelint/config',
+  ],
+
+  // conflicts with prettier formatting
+  rules: {
+    'scss/double-slash-comment-empty-line-before': null,
```

Comment:
Rules from [`stylelint-scss`](https://github.com/kristerkari/stylelint-scss) don't seem to have fixers to take advantage of. I'm pretty sure we'll see other rules.